Assignments/assignment8/src/even_odd.rs

```diff
@@ -0,0 +1,30 @@
+// even response return results weather a no.is even or not.
+//
+// #Argument
+// number-type i32
+//
+// #Return
+// String.
+pub fn even_response(number: i32) -> String {
+    let result = even_calculator(number);
+    match result {
+        Ok(_number) => ("no is even").to_string(),
+        Err(_msg) => ("Oh no is not even").to_string(),
+    }
+}
+
+// even_calculator check weather a no.is even or not.
+//
+// #Argument
+// number-type i32
+//
+// #Return
+// <String, String>-Type result
```

Comment:
Describe a little bit more for this Return like generic type or error handling type.